pkgs/development/python-modules/pydevd/default.nix

```diff
@@ -48,6 +49,9 @@ buildPythonPackage rec {
     # AssertionError pydevd_tracing.set_trace_to_threads(tracing_func) == 0
     "test_tracing_other_threads"
     "test_tracing_basic"
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
Thank you for testing on darwin and sorry for my late reply. When it's possible you can add a comment on why it is failing or maybe link to an upstream issue

Reply:
if you let me know what would be a good summary in your opinion, I'm happy to add a comment

Reply:
Yeah this looks complicated and unfortunately I'm not into pydevd